.travis.yml

```diff
@@ -22,10 +22,17 @@ matrix:
       env: SYMFONY_VERSION='2.5.*'
     - php: 5.3.3
       env: DEPENDENCIES='low'
+    - php: 5.6
+      env: DEPENDENCIES='dev'
+    - php: 5.6
+      env: DEPENDENCIES='dev' SYMFONY_VERSION="3.0.*@dev"
```

Comment:
``DEPENDENCIES='dev'`` already installs the 3.0 versions of Symfony as this is the highest compatible version (except for symfony/process because you forgot to change the requirement). So it is useless to add a second job